deploy/charts/rook-ceph/templates/aggregate-roles.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.rbacAggregate.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Are these roles automatically added to any k8s user in the view/edit group, or are they only enabled for users after more admin control? I wonder if it could be easier to include these as default roles created by Rook, which would not add complexity to the helm chart. As long as doing so wouldn't accidentally give users extra permissions, maybe we should consider that.

Reply:
the roles are added to the [cluster roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)  view / edit / admin respectively  thanks to the [ClusterRole aggregation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles).

view/edit/admin are default roles of the ClusterRole aggregator.

then a "human" admin binds those role to users in a namespace (using rolebinding), so it's disabled by default to ensure backward compatibility.